.github/workflows/nightly.yml

```diff
@@ -28,69 +28,131 @@ jobs:
     name: "Pre-compile Checks"
     uses: ./.github/workflows/template.pre-compile-checks.yml
 
-  java8:
-    name: "Java 8"
-    uses: ./.github/workflows/template.flink-ci.yml
-    with:
-      workflow-caller-id: java8
-      environment: 'PROFILE="-Dinclude_hadoop_aws"'
-      jdk_version: 8
-    secrets:
-      s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
-      s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
-      s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
-  java11:
-    name: "Java 11"
-    uses: ./.github/workflows/template.flink-ci.yml
-    with:
-      workflow-caller-id: java11
-      environment: 'PROFILE="-Dinclude_hadoop_aws -Djdk11 -Pjava11-target"'
-      jdk_version: 11
-    secrets:
-      s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
-      s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
-      s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
-  java17:
-    name: "Java 17"
-    uses: ./.github/workflows/template.flink-ci.yml
-    with:
-      workflow-caller-id: java17
-      environment: 'PROFILE="-Dinclude_hadoop_aws -Djdk11 -Djdk17 -Pjava17-target"'
-      jdk_version: 17
-    secrets:
-      s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
-      s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
-      s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
-  java21:
-    name: "Java 21"
-    uses: ./.github/workflows/template.flink-ci.yml
-    with:
-      workflow-caller-id: java21
-      environment: 'PROFILE="-Dinclude_hadoop_aws -Djdk11 -Djdk17 -Djdk21 -Pjava21-target"'
-      jdk_version: 21
-    secrets:
-      s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
-      s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
-      s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
-  hadoop313:
-    name: "Hadoop 3.1.3"
-    uses: ./.github/workflows/template.flink-ci.yml
-    with:
-      workflow-caller-id: hadoop313
-      environment: 'PROFILE="-Dflink.hadoop.version=3.2.3 -Phadoop3-tests,hive3"'
-      jdk_version: 8
-    secrets:
-      s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
-      s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
-      s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
-  adaptive-scheduler:
-    name: "AdaptiveScheduler"
-    uses: ./.github/workflows/template.flink-ci.yml
-    with:
-      workflow-caller-id: adaptive-scheduler
-      environment: 'PROFILE="-Penable-adaptive-scheduler"'
-      jdk_version: 8
-    secrets:
-      s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
-      s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
-      s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
+#  java8:
```

Comment:
Is this code no longer needed? Why not just delete it?

Reply:
That's just disabled code to fasten the test run. It will be reverted before merging the PR.

Reply:
Yes, I will put it back once we finalize the PR, commented just to run to create the wheels.